terraform/gcp/io.tf

```diff
@@ -0,0 +1,46 @@
+variable "role_name" {
+  type        = string
```

Comment:
why not use the default?

Reply:
How do you mean?

Reply:
Don't need to define type if setting a default, i think? Maybe even when not setting default

Reply:
Ah I see, you're right. Just being more explicit)